index.js

```diff
@@ -7,12 +7,37 @@ function ask(questionText) {
   });
 }
 
+let loNum = 0;
+let hiNum = 100;
+let yesOrNo = "n";
+let numTries = 0;
```

Comment:
You can initialize these variables inside the start function instead of here to get rid of global variables